WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1155,6 +1155,16 @@ FormAddRemoveButton          = @FormButtonHeight@
 FormButtonStyle              = @cssShadow@:0px 0px 0px transparent; padding:5px; @ActiveCursor@ @PanelBorder@ @MediumRoundCorners@ background-color:@WidgetBgColor@; @ButtonText@
 
 ###################
+#   SHORTCUT LIST
```

Comment:
It is not a recommendation to only use one unit. I chose to use value's that made more sense considering the impact. For ex: If we want to have fixed margin irrespective of zoom or parent font-size : use px. 
If we want to have breathing space's increased on zoom or with increase in parent font-size use em/rem.